### PR TITLE
Release 1.11.0

### PR DESCRIPTION
**Pull request by Rashmil-1999**
### Changed
- New IN-CORE logo and color palette [#193](https://github.com/IN-CORE/incore-ui/issues/193)

### CODE REVIEW

**Comment by ylyangtw** [comment]
Looks good. But I saw incore-docs repo's release date is 2024-05-02, not sure whether it matters.

**Review by ylyangtw — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [1.11.0] - 2024-04-26
```
Comment (longshuicy) [inline CHANGELOG.md:9]: @ylyangtw is right about the date. Could you change that to next Thursday?

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (longshuicy) [reply CHANGELOG.md:9]: I think we decided to change this to 4/30 next Tuesday instead.

Comment (Rashmil-1999) [reply CHANGELOG.md]: updated. :)

**Review by longshuicy — APPROVED** [review]

**Pull request merged**